tests/tests/test_monitor_client.py

```diff
@@ -1187,10 +1189,84 @@ def test_monitorclient_logs_and_services(self, monitor_commercial_setup_no_clien
         assert fds_count1.rstrip().isnumeric()
         fds_count1 = int(fds_count1.rstrip())
         logger.info(
-            "test_monitorclient_alert_email: currently %s fds open" % fds_count0
+            "test_monitorclient_logs_and_services: currently %s fds open" % fds_count0
         )
         assert abs(fds_count1 - fds_count0) <= max_fds_count_diff
-        logger.info("test_monitorclient_alert_email: fds count have not increased")
+        logger.info(
+            "test_monitorclient_logs_and_services: fds count have not increased"
+        )
+
+    def test_monitorclient_logs_and_patterns(self, monitor_commercial_setup_no_client):
+        """Tests the monitor client email alerting for multiple patterns types"""
+        mailbox_path = "/var/spool/mail/local"
+        wait_for_alert_interval_s = 8
+        expected_from = "noreply@mender.io"
+        user_name = "some.user+{}@example.com".format(str(uuid.uuid4()))
+        devid, _, auth, mender_device = self.prepare_env(
+            monitor_commercial_setup_no_client, user_name
+        )
+        inventory = Inventory(auth)
+        logger.info("test_monitorclient_logs_and_patterns: env ready.")
+
+        logger.info(
+            "test_monitorclient_logs_and_patterns: email alert on log file containing a pattern scenario."
+        )
+        log_file = "/tmp/mylog.log"
+        log_pattern = "session opened for user \\w+"
```

Comment:
Should this be:
```suggestion
        log_pattern = "session opened for user \w+"
```
?

Reply:
I actually paused the test, and verified, that it is correctly present in the check file: 
```bash
2021-09-27 08:07:44 [INFO] [ctl.sh:cleanup:15] exiting.
root@qemux86-64:~# cat /etc/mender-monitor/monitor.d/available/log_mylog-pcre.sh
# This file was autogenerated by Monitoring Utilities based on the configuration
SERVICE_NAME="mylog-pcre"
LOG_PATTERN="session opened for user \w+"
LOG_FILE="@tail -f /tmp/mylog.log"
```

the single \ in the double quotes will get interpreted, while \\ will be verbatim \
in this case you maybe right, since '\w' is not a valid sequence (as \n is) but I believe it is "more correct" to use \\ in this case, since we want "\w" in the string